### PR TITLE
refactor(ValidFieldText): replace tel to float in type prop but still use tel under the hood

### PR DESCRIPTION
**Pull request by rayhan-ab**

### CODE REVIEW

**Comment by rayhan-ab** [comment]
it is intended i don't bump the version so we only need to bump it once after @hamdanhanafi's PR is merged

**Comment by hamdanhanafi** [comment]
give me context, why we should rename prop from tel to float ?

**Comment by rayhan-ab** [comment]
> give me context, why we should rename prop from tel to float ?

https://bitbucket.org/accelbyte/justice-adminportal-website/pull-requests/3032#Lmodules/ap-storage-statistic-management/src/components/ConfigurationsPage/CreateStatisticConfigurationModal.tsxT458

well yeah, using type="float" is more intuitive than type="tel" + isFloat=true @hamdanhanafi

File: src/components/ValidatedInput/ValidFieldText.tsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ export interface ValidFieldTextProps
   dataQaProps?: string | null;
   validFieldTextRef?: React.RefObject<HTMLDivElement>;
   rightIcon?: React.ReactNode;
+  /** @deprecated Use type="float" instead */
   isFloat?: boolean;
```
Comment (aziz-zen-ab) [inline src/components/ValidatedInput/ValidFieldText.tsx:54]: Is there any field using the `isFloat` props right now? I don't find any in AP master, how about delete it here?

Comment (rayhan-ab) [reply src/components/ValidatedInput/ValidFieldText.tsx:54]: okay

Comment (rayhan-ab) [reply src/components/ValidatedInput/ValidFieldText.tsx]: done

**Comment by Manuel-AB** [comment]
@rayhan-ab please build this PR first

> more intuitive than type="tel" + isFloat=true

what if the type is still `number` but set `isFloat` to `true`? Therefore, the type in the background changes from `number` to `tel`

**Comment by rayhan-ab** [comment]
it's already built, mate.
we no longer use isFloat as per @aziz-zen-ab's suggestion
@Manuel-AB

**Review by Manuel-AB — APPROVED** [review]

**Pull request merged**